reana_server/rest/workflows.py

```diff
@@ -1,56 +1,53 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of REANA.
-# Copyright (C) 2018, 2019, 2020, 2021, 2022 CERN.
+# Copyright (C) 2018, 2019, 2020, 2021, 2022, 2023 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Reana-Server workflow-functionality Flask-Blueprint."""
-import os
 import json
 import logging
+import os
 import traceback
 
 import requests
 from bravado.exception import HTTPError
-from flask import Blueprint, Response
-from flask import jsonify, request, stream_with_context
+from flask import Blueprint, Response, jsonify, request, stream_with_context
 from jsonschema.exceptions import ValidationError
-
 from reana_commons import workspace
 from reana_commons.config import REANA_WORKFLOW_ENGINES
 from reana_commons.errors import REANAQuotaExceededError, REANAValidationError
+from reana_commons.specification import load_reana_spec
 from reana_commons.validation.operational_options import validate_operational_options
 from reana_commons.validation.utils import validate_workflow_name
-from reana_commons.specification import load_reana_spec
 from reana_db.database import Session
 from reana_db.models import InteractiveSessionType, RunStatus
 from reana_db.utils import _get_workflow_with_uuid_or_name
-from webargs import fields, validate
-from webargs.flaskparser import use_kwargs
-
 from reana_server.api_client import current_rwc_api_client
 from reana_server.config import REANA_HOSTNAME
 from reana_server.decorators import check_quota, signin_required
 from reana_server.deleter import Deleter, InOrOut
-from reana_server.validation import (
-    validate_inputs,
-    validate_workspace_path,
-    validate_workflow,
-)
 from reana_server.utils import (
-    _fail_gitlab_commit_build_status,
     RequestStreamWithLen,
-    _load_and_save_yadage_spec,
+    _fail_gitlab_commit_build_status,
     _get_reana_yaml_from_gitlab,
-    prevent_disk_quota_excess,
-    publish_workflow_submission,
+    _load_and_save_yadage_spec,
     clone_workflow,
     get_quota_excess_message,
     get_workspace_retention_rules,
     is_uuid_v4,
+    prevent_disk_quota_excess,
+    publish_workflow_submission,
 )
+from reana_server.validation import (
+    validate_inputs,
+    validate_workflow,
+    validate_workspace_path,
+)
+from webargs import fields, validate
+from webargs.flaskparser import use_kwargs
```

Comment:
As for r-wokflow-controller, please check the order of imports https://github.com/reanahub/reana/wiki/Tips-for-Python#imports

Reply:
I'm pretty sure it's correct but I'm a bit short on time. Please double check.

The change has been applied to this superseding PR: #658